x-pack/plugins/alerting/server/routes/lib/rewrite_actions.ts

```diff
@@ -15,9 +17,15 @@ export const rewriteActionsReq = (
 ): NormalizedAlertAction[] => {
   if (!actions) return [];
 
-  return actions.map(({ frequency, alerts_filter: alertsFilter, ...action }) => {
+  return actions.map((action) => {
+    if (action.type === RuleActionTypes.SYSTEM) {
```

Comment:
`isSystemAction` does not work with `NormalizedAlertAction`. I could make it work but I need to import `NormalizedAlertAction` in the `is_system_action.ts` file which is in `common` and cannot import from the `server/*` folder. We can export `NormalizedAlertAction` from `common` but It seems like an implementation detail of the `server` and not something we would like to export. If you feel differently let me know.